### ISSUE TITLE
Enable WriteRead*MissingPerformGets` tests

### ISSUE THREAD

**Issue by vicentebolea** [ornladios/ADIOS2#3725]
In #3722 we disabled WriteRead*MissingPerformGets` tests since they were disabled before this big refactoring but after running clang-format they got re-enabled as the gtest TEST_F does not work if its expanded in a multi-line invocation as it was before this PR. This needs to be resolved.

Enabling these tests results errors when BP5 is used.

_Originally posted by @vicentebolea in https://github.com/ornladios/ADIOS2/issues/3722#issuecomment-1656318408_

**Comment by vicentebolea** [comment]
They were disabled at least since 2018 and edited in this commit: e2f7f7ea6c

**Comment by vicentebolea** [comment]
@eisenhauer

**Comment by eisenhauer** [comment]
Yeah, these tests need to be refactored before they'll work with BP5 as they rely upon BP4 behaviour that we explicitly deprecated.  Commenting out for now is fine and I'll clean up after this is merged.  (There are probably more tests that got disabled at some point and should be re-enabled, so a broader look might be useful.)

**Comment by vicentebolea** [comment]
This tests has been re-enabled
